### ISSUE TITLE
Biome Compatibility

### ISSUE THREAD

**Issue by WolfieWaffle** [MCUmbrella/ATG-Modified#9]
It appears that biome mods do not work with this version. Is this just because that hasn't been added yet? Also, Is this project still being developed?

**Comment by MCUmbrella** [comment]
Yes this mod only support vanilla biomes for now. Due to time and technical reasons this mod will not be updated in a few months.
Anyway, thank you for your support

**Comment by MagmaSlime123** [comment]
4 years later. Dead mod.
